db/migrate/20161202161114_add_files_collection_ref_to_tribunal_case.rb

```diff
@@ -0,0 +1,5 @@
+class AddFilesCollectionRefToTribunalCase < ActiveRecord::Migration[5.0]
+  def change
+    add_column :tribunal_cases, :files_collection_ref, :uuid, default: 'uuid_generate_v4()', unique: true
```

Comment:
I'm happy with this but wasn't the original idea that the ID would be generated in the uploader the first time you uploaded something? We're kinda duplicating that here – but then again to me it makes sense to generate an ID here and the uploader takes arbitrary IDs anyway... @digitalronin